pkg/cmd/install.go

```diff
@@ -74,6 +74,7 @@ func newCmdInstall(rootCmdOptions *RootCmdOptions) *cobra.Command {
 	cmd.Flags().StringVar(&impl.buildStrategy, "build-strategy", "", "Set the build strategy")
 	cmd.Flags().StringVar(&impl.buildTimeout, "build-timeout", "", "Set how long the build process can last")
 	cmd.Flags().StringVar(&impl.traitProfile, "trait-profile", "", "The profile to use for traits")
+	cmd.Flags().BoolVar(&impl.cache, "cache", true, "To enable or disable the cache in building phase")
```

Comment:
'cache' is perhaps a bit too vague as a flag name. Maybe align it with the other build related flags. E.g `build-cache`.

Reply:
thanks for the review, I have updated it as per your suggestions